contract-map.json

```diff
@@ -230,6 +246,23 @@
     "decimals": 18,
     "symbol": "META"
   },
+  "0x9a642d6b3368ddc662CA244bAdf32cDA716005BC": {
+    "name": "Qtum",
+    "logo": "qtum_28.png",
+    "erc20": true,
+    "decimals": 18,
```

Comment:
According to Etherscan, Qtum has 8 decimals. Can you get a source to corroborate this? https://etherscan.io/token/0x697beac28b09e122c4332d163985e8a73121b97f

Reply:
That is QRL, Qtum is

https://etherscan.io/token/Qtum#readContract

Reply:
Got it, confirmed.